roles/cfg_openwrt/templates/common/config/network.j2

```diff
@@ -39,6 +39,8 @@ config interface '{{ name }}'
     {% if network.get('enforce_client_isolation') and role == 'corerouter' and
           not bridge_needed %}
 	option macaddr '02:00:00:00:00:01'
+    {% elif network.get('mac_override') and role == 'corerouter' %}
```

Comment:
why are you implementing this? Please open seperate PR if you wanna introduce a feature